### PR TITLE
Adds page routes, actions, pages and styles.

### PR DESCRIPTION
**Pull request by arothuis**
[Ticket 106410466](https://www.pivotaltracker.com/story/show/106410466)

Screenshot:
screen shot 2015-10-27 at 16 05 06

### CODE REVIEW

**Comment by arothuis** [comment]
@rjkip :chicken:

File: src/OpenConext/ProfileBundle/Resources/config/routing.yml [hunk 1]
```diff
@@ -1,6 +1,18 @@
-profile_homepage:
+profile_index:
     path:     /
-    defaults: { _controller: OpenConextProfileBundle:Default:index }
+    defaults: { _controller: OpenConextProfileBundle:Page:index }
+
+profile_my_profile:
+    path:     /my-profile
```
Comment (rjkip) [inline src/OpenConext/ProfileBundle/Resources/config/routing.yml]: Why two URLs and actions? I'd merge these into `/` and `myProfileAction`. Then you can use `app.request.attributes.get('_route')` to activate navigation links.

File: src/OpenConext/ProfileBundle/Controller/PageController.php [hunk 2]
```diff
@@ -20,10 +20,25 @@
 
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 
-class DefaultController extends Controller
+class PageController extends Controller
```
Comment (rjkip) [inline src/OpenConext/ProfileBundle/Controller/PageController.php]: Please split into separate, meaningful controllers like we discussed IRL. Change route names accordingly.

**Comment by rjkip** [comment]
Lookin' slick. :package:

**Pull request merged**